### PR TITLE
Add hiring rate to about page

### PR DESCRIPTION
**Pull request by sofer**

### CODE REVIEW

**Comment by sofer** [comment]
@izaakrogan I have updated the hiring rate. On the measure used by other schools, i.e. *"job-seeking graduates"*, our hiring rate is a solid 100% and has been since the first course we ran in January 2014.

File: _harp/about/_company.md [hunk 1]
```diff
@@ -9,3 +9,14 @@ We support ourselves entirely through voluntary contributions from our alumni, t
 We believe this to be a replicable model of training that is capable of addressing acute skills shortages, widening access to work opportunities, and offering life-changing personal empowerment free at the point of delivery and without ongoing subsidies.
 
 We are partnering with a number of organisations in the social enterprise and charity sectors to provide freelancing and employment opportunities for our graduates and to open new schools in the UK and abroad.
+
+## Our hiring rate
+
+We encourage our graduates to freelance with us after they graduate, so the usual measure of graduate success, the *"hiring rate"*,  a bit misleading, but of the 52 students who studied with us in 2014/15 and who actively sought development work, all 52 are working either freelance or in permanent roles. In effect, our success rate is 100%.
```
Comment (iteles) [inline _harp/about/_company.md]: I think this is too wordy. I would just go for one sentence: Our hiring rate for our graduates who have *actively sought* full time employment after the course is 100%.

**Comment by sofer** [comment]
@iteles thank you for your suggestion. How's this?

**Comment by sofer** [comment]
@izaakrogan or @iteles please do check and merge when you get a minute.

File: _harp/about/_company.md [hunk 2]
```diff
@@ -9,3 +9,17 @@ We support ourselves entirely through voluntary contributions from our alumni, t
...
 
 We are partnering with a number of organisations in the social enterprise and charity sectors to provide freelancing and employment opportunities for our graduates and to open new schools in the UK and abroad.
+
+## Our hiring rate
+
+Our employment success rate for students who have actively sought development work after graduation is 100%.
+
+Of the 52 students who studied with us in 2014/15 and who actively sought development work, all 52 are now working either freelance or in permanent roles. This is a reflection of both the quality of our graduates and the level of support we provide to them after they graduate.
+
+Meanwhile, the proportion of students who have actually decided to seek development work after they graduate has grown dramatically for us in the last two years. The transformation from only one student in our very first (FAC0) cohort to every student in the Autumn 2015 (FAC6) cohort is a reflection of the growing reputation of the programme and the serious intent of our students.
```
Comment (iteles) [inline _harp/about/_company.md]: I'm not convinced this (and the graph) actually adds any value. What matters is that right now, 100% of the people who seek employment get it. The others become freelancers or start their own business. 

We're _Founders_ & Coders. Does it matter whether people are becoming developers? Also, this isn't strictly true, Kat is starting her own business and doing dev on the side and for a short period of time.

Comment (sofer) [inline _harp/about/_company.md]: But doesn't the graph tell a nice story about how the course outcomes have changed over the last two years?

Comment (iteles) [inline _harp/about/_company.md]: Maybe I'm just not getting this importance from the paragraph that explains it. I think it might be the 'seek development work' phrase - I reckon it's just not clear what that means.
Maybe 'decided to work as web developers after they graduate has grown dramatically in the last two years' ('seek' is something I use for only job-seekers. Might be a language barrier thing.

File: _harp/about/_company.md [hunk 3]
```diff
@@ -9,3 +9,17 @@ We support ourselves entirely through voluntary contributions from our alumni, t
 We believe this to be a replicable model of training that is capable of addressing acute skills shortages, widening access to work opportunities, and offering life-changing personal empowerment free at the point of delivery and without ongoing subsidies.
 
 We are partnering with a number of organisations in the social enterprise and charity sectors to provide freelancing and employment opportunities for our graduates and to open new schools in the UK and abroad.
+
+## Our hiring rate
+
+Our employment success rate for students who have actively sought development work after graduation is 100%.
+
+Of the 52 students who studied with us in 2014/15 and who actively sought development work, all 52 are now working either freelance or in permanent roles. This is a reflection of both the quality of our graduates and the level of support we provide to them after they graduate.
```
Comment (iteles) [inline _harp/about/_company.md]: Isn't this first sentence the same as the sentence above?

Comment (sofer) [inline _harp/about/_company.md]: Removed one of the paragraphs and changed "development work" to "work as a web developer"

**Comment by izaakrogan** [comment]
I really like how Thinkful have abstracted all the details into a report, here: https://www.thinkful.com/bootcamp-jobs-stats/audit-results-feb-2016.pdf

Then they have liberty to go about saying **100% THIS** and **95% THAT** without having to qualify statim - and create overhead/clutter in their copy.

Can we stick all this on a separate page on the site and link to it. Happy to do this if useful
